spotirec/oauth2.py

```diff
@@ -19,9 +19,10 @@ def __init__(self):
         self.client_id = '466a89a53359403b82df7d714030ec5f'
         self.client_secret = '28147de72c3549e98b1e790f3d080b85'
         self.redirect = f'http://localhost:{self.PORT}'
-        self.scopes = 'user-top-read playlist-modify-public playlist-modify-private ' \
-                      'user-read-private user-read-email ugc-image-upload ' \
-                      'user-read-playback-state user-modify-playback-state user-library-modify'
+        self.scopes = ['user-top-read', 'playlist-modify-public', 'playlist-modify-private',
+                       'user-read-private', 'user-read-email', 'ugc-image-upload',
+                       'user-read-playback-state', 'user-modify-playback-state',
+                       'user-library-modify', 'user-library-read']
```

Comment:
Does this new scope require reauth? Afair this happens automatically now, right?

Reply:
Yep, found it just a page down in changes :+1: